master/internal/experiment.go

```diff
@@ -202,8 +217,11 @@ func (e *experiment) Receive(ctx *actor.Context) error {
 		ops, err := e.searcher.TrialCreated(msg.create, msg.trialID)
 		e.processOperations(ctx, ops, err)
 	case trialReportValidation:
-		ops, err := e.searcher.ValidationCompleted(msg.trialID, msg.metrics)
+		ops, err := e.searcher.ValidationCompleted(msg.trialID, msg.metric)
 		e.processOperations(ctx, ops, err)
+		if ctx.ExpectingResponse() {
+			ctx.Respond(nil)
+		}
```

Comment:
the system returns `actors.errNoResponse` once it is done processing the ask but since that isn't public I can't check it.. I could make it public but I preferred just to add a response. Maybe i'll make it public and check for ErrNoResponse

Reply:
Eh.  It's prolly fine.